README.md

```diff
@@ -1,18 +1,27 @@
-# Краткая инструкция по работе
+# Work instructions 👩‍🎓👨‍🎓
```

Comment:
1. Replace title with name of our project
2. Emoji should be written with shortcode (https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md)
3. After the title, write a short description about this project (what is it, why are we doing it, etc.)